src/caret_analyze/record/records_service/response_time.py

```diff
@@ -299,13 +308,14 @@ def to_worst_with_external_latency_case_records(
                                                          key=lambda x: x[0]):
             if converter:
                 record = {
-                    self._start_column: round(converter.convert(start_ts - worst_to_best_ts)),
-                    'response_time': end_ts - (start_ts - worst_to_best_ts)
+                    self._start_column: round(
+                        converter.convert(max(start_ts - worst_to_best_ts, 0))),
+                    'response_time': max(end_ts - (start_ts - worst_to_best_ts), 0)
```

Comment:
From my understanding, we need to print warning and ignore data when its response time is lesser than 0. Can you modify this change and another changes in this file according to this policy?

Reply:
Fixed in [cc43ec7](https://github.com/tier4/caret_analyze/pull/455/commits/cc43ec754a8a2f0cc4fc2f14d9479bde3042e19b).

Reply:
I think current code does not follow the policy I mentioned above. (There is no value checking and warning printing.)
Can you check the code again?